owners/edf-methanesat-ee.jsonnet

```diff
@@ -1,5 +1,5 @@
 {
-  "contactDisplay": "support@methanesat.org",
```

Comment:
You are fixing this in the other PR, so this file can be reverted

Reply:
yes.  Removed this file from the PR